brianlong/www.validators.app
# 181819673 Update ruby to 2.7

#### What's this PR do?
- Upgrades Ruby version in app

#### How should this be manually tested?
- It's best to test it on staging, because all scripts and crons are turned on there
- Click through the app, confirm that nothing crashes
- Confirm that all scripts, workers etc. are working
- Check appsignal and confirm that there're no new errors

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/181819673)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
